src/oca/go/.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+sudo: false
+
+go:
+    - 1.8
+
+branches:
+  only:
+  - master
+
+before_install:
+  # Install Glide
+  - mkdir -p "${GOPATH}/bin"
+  - curl https://glide.sh/get | bash
+
+  # Get deps
+  - go get github.com/mattn/goveralls
+  - go get -u github.com/golang/lint/golint
+  - make get-deps
+
+script:
+  - make test
+  - $GOPATH/bin/goveralls -service=travis-ci
```

Comment:
we should remove this as we are not using Travis for CI

Reply:
It's already deleted.